### PR TITLE
GPII-3927: Office simplification

### PR DESCRIPTION
**Pull request by stegru**
This contains the code to update the ribbon on Office applications.

@Karadaliev, I've made enough progress on the office simplification for you to start working with it. At the moment, it only performs the file swapping and doesn't restart Word.

If you point your reference of gpii-windows to this branch, `stegru/windows#GPII-3927`, it will pull in the branch of universal I'm working on.

I'm guessing this will be implemented in a similar way that the high-contrast or screen-scale widgets speak with core, the preference is `http://registry\.gpii\.net/applications/com\.microsoft.office` (the final `.` is not escaped).

The values can be one of the following, depending on the checkbox values:
`StandardSet`
`Basics+StandardSet`
`Essentials+StandardSet`
`Basics+Essentials+StandardSet`

(if that's tricky to implement in the widgets, another method can be used - such as 0-4).

### CODE REVIEW

**Comment by amb26** [comment]
@stegru - don't forget to make the matching pull against universal!

**Comment by stegru** [comment]
done. Ready for review

**Comment by stegru** [comment]
@karadaliev - it's now complete.

**Comment by stegru** [comment]
ok to test

```
Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).

Failures
 - nodejs.install - nodejs.install not installed. The package was not found with the source(s) listed.
```

**Comment by stegru** [comment]
ok to test

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "dependencies": {
         "edge-js": "10.3.1",
         "ffi-napi": "2.4.3",
-        "gpii-universal": "0.3.0-dev.20190529T130102Z.a3fea390",
+        "gpii-universal": "stegru/universal#GPII-3927",
```
Comment (amb26) [inline package.json]: This got merged as far as I can see - can you update to latest release of 0.3.0-dev.20190801T142013Z.3e58c798 and see if it works?

Comment (stegru) [reply package.json]: done

**Pull request merged**